test/test_css_parser_offset_capture.rb

```diff
@@ -27,10 +27,16 @@ def test_capturing_offsets_for_local_file
     assert_equal file_name, rules[1].filename
   end
 
+  def fixture(*path)
```

Comment:
maybe make this global in the helper.rb somewhere

Reply:
donw